spp-001-nl mhq heide (#69)
inbo/protocolsource: source/spp/mhq/spp_001_nl_mhq_heide/04_werkwijze.Rmd

De doelstelling is om de subprotocols te organiseren in een coherent hoofdprotocol waarin ontbrekende elementen aangevuld worden die niet in de subprotocols staan, zoals wanneer een bepaald subprotocol wel of niet wordt uitgevoerd, of kleine extra stappen die geen eigen subprotocol vereisen.
-->
```
Volg volgende stappen in chronologische volgorde:

> 1.  navigatie naar meetpunt
>
> 2.  Vegetatieplot installeren
>
> 3.  fototografisch documenteren van het meetpunt
>
> 4.  bepalen of een meetpunt tot de doelpopulatie behoort of niet
>
> 5.  eventueel vervangen van de meetlocatie in het geval het originele meetpunt niet tot de doelpopulatie behoort
>
> 6.  subprotocol vegetatie-opname met Londo-schaal
>
> 7.  meten van de variabelen in de 18 meter structuurplot

### STAP 1: Navigatie naar een meetpunt

De navigatie naar de steekproefpunten gebeurt met behulp van een zogeheten RTK-GNSS of FIELDMAP installatie (of combinatie van beiden) waarin de X en Y coördinaten van de proefvlakken op voorhand ingelezen zijn.
De nauwkeurigheid van een commerciële GPS of een ingebouwde GPS in een smartphone is te laag om bij een herbezoek hetzelfde proefvlak voldoende goed te lokaliseren.

> Het kan wel handig zijn om met zo'n toestel eerst naar de omgeving van het meetpunt te navigeren en pas dan over te schakelen naar de nauwkeurige GPS-apparatuur voor het inmeten van de exacte locatie.Eens het meetpunt bepaald is, wordt het vegetatieproefvlak van 3 op 3 meter geïnstalleerd in stap 2.

Reviewer: ```suggestion
Het kan wel handig zijn om met zo'n toestel eerst naar de omgeving van het meetpunt te navigeren en pas dan over te schakelen naar de nauwkeurige GPS-apparatuur voor het inmeten van de exacte locatie.
Eens het meetpunt bepaald is, wordt het vegetatieproefvlak van 3 op 3 meter geïnstalleerd in stap 2.
```

### STAP 2: Installeren van het vegetatieproefvlak (3 m x 3 m)

> Het vegetatieproefvlak wordt geïnstalleerd volgens de werkwijze in [protocol sfp-401-nl versie 2023.03](../2023.03/index.html)

Reviewer: ```suggestion
Het vegetatieproefvlak wordt geïnstalleerd volgens de werkwijze in [protocol sfp-401-nl versie 2023.03](../2023.03/index.html).
```

De oriëntatie van het proefvlak is als volgt:

> -   het punt waar naartoe genavigeerd werd is de zuidoostelijke hoek van de vegetatieplot. Hier wordt er een eerst hoekpaaltje geplaatst en van daaruit wordt met behulp van een (digitaal) kompas 3 meter naar het noorden afgemeten en een tweede paaltje geplaatst. Dit wordt herhaal vanaf het tweede hoekpunt naar het westen en tenslotte vanuit het derde hoekpunt naar het zuiden, zodat een vierkant met oppervlakte 9m² bekomen wordt (zie Figuur \@ref(fig:plot9m))

Reviewer 2: ```suggestion
-   het punt waar naartoe genavigeerd werd is de zuidoostelijke hoek van de vegetatieplot. Hier wordt er een eerst hoekpaaltje geplaatst en van daaruit wordt met behulp van een (digitaal) kompas 3 meter naar het noorden afgemeten en een tweede paaltje geplaatst. Dit wordt herhaald vanaf het tweede hoekpunt naar het westen en tenslotte vanuit het derde hoekpunt naar het zuiden, zodat een vierkant met oppervlakte 9m² bekomen wordt (zie Figuur \@ref(fig:plot9m))
```

> (ref:plot9m) Oriëntatie van de vegetatie-en structuurplot

Reviewer 2: ```suggestion
(ref:plot9m) Oriëntatie van de vegetatie- en structuurplot
```

```{r plot9m, fig.cap="(ref:plot9m)", out.width="75%", fig.align ="center"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "plot9M.png"))
```

### STAP 3: Fotografisch documenteren

Neem de foto's vooraleer je de vegetatie-opname maakt.
Foto's worden altijd gemaakt, ook wanneer er geen opname zal uitgevoerd worden (zie 5.1.5).

> Indien je werkt met een externe camera codeer het beeld dan duidelijk in de daartoe voorziene velden in de veldcomputer of het vrije veld 'opmerkingen'van de papieren formulieren.

Reviewer 2: ```suggestion
Indien je werkt met een externe camera codeer het beeld dan duidelijk in de daartoe voorziene velden in de veldcomputer of het vrije veld 'opmerkingen' van de papieren formulieren.
```

Je kan ook eerste een foto maken van het unieke ID nummer van de plot en daarna de foto's van de vegetatie-opname maken.
De foto's worden genomen vanuit het zuiden kijkend naar het noorden.
De camera tilten in portrait oriëntatie waardoor de vegetatie maximaal in het fotokader valt.

> Neem 2 foto's per meetpunt: één overzichts foto waarbij je de plot landschappelijk kadert, en een tweede waarbij je enkel de vegetatie in de plot fotografeert (zie Figuur \@ref(fig:fotovoorbeeld)).

Reviewer 2: ```suggestion
Neem 2 foto's per meetpunt: één overzichtsfoto waarbij je de plot landschappelijk kadert, en een tweede waarbij je enkel de vegetatie in de plot fotografeert (zie Figuur \@ref(fig:fotovoorbeeld)).
```

(ref:fotovoorbeeld) Voorbeeld van een foto van het proefvlak in het landschap (links) en ingezoomd op het proefvlak zelf (rechts)

```{r fotovoorbeeld, fig.cap="(ref:fotovoorbeeld)"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "fotoproefvlak.png"))
```

### STAP 4: Bepalen of een meetpunt tot de doelpopulatie behoort of niet

> In deze stap dienen we te controleren of de doelvegetatie effectief aanwezig is in het vegetatieproefvlak opdat we een meting kunnen uitvoeren.

Reviewer: ```suggestion
Vooraleer we een meting uitvoeren, moeten we eerst controleren of de doelvegetatie effectief aanwezig is in het vegetatieproefvlak.
```

    -   --\> opname doen wanneer doelvegetatie in de vegetatieplot gelijk of groter is dan 50%
-   de structuurplot bevat doelvegetatie en een andere vegetatietype(s) in mozaïek of in overgang met elkaar
    -   --\> opname niet doen wanneer doelvegetatie inclusief overgangen minder is dan 50%, ga naar stap 5 om te checken of het meetpunt in aanmerking komt voor opname bij een ander type
    -   --\> opname doen wanneer doelvegetatie inclusief overgangen gelijk of meer is dan 50% bedraagt. Overgangen tussen de verschillende heidehabitats zijn ruim te interpreteren (bv. snavelbiesverbond gaat snel over in 4010, 2330_bu en 2310 vormen mozaïeken, 4010 met 7140_oli, enz.)

Indien het eerder een geleidelijke overgang betreft naar een verwant vegetatietype dan wordt dit beschouwd als een onderdeel van de doelvegetatie en wordt de opname gemaakt inclusief de overgang.

Noteer steeds op welk aandeel (in %) van de plot je data betrekking heeft.
Bij het schatten blijf je echter werken alsof je op een volledige plot zou doen, je gaat m.a.w. niet herrekenen voor het aandeel dat effectief doelvegetatie bevat.

> In een 3mx3m plot blijft 30cmx30cm dus 1%.

Reviewer 2: ook elders nakijken en verbeteren

Voor de structuurplot (zie stap 7) wordt gelijkaardig gewerkt en dient ook het aandeel doelvegetatie waarop de inschatting betrekking heeft vermeld te worden.
De inschatting van de bedekking gebeurt eveneens alsof het een volledige cirkelplot betreft, m.a.w. 10 m² is 1%.

> In geval van twijfel voeren we de opname uit en documenteren we in het opmerkingenveld

Reviewer: ```suggestion
In geval van twijfel voeren we de opname uit en documenteren we in het opmerkingenveld.
```

### STAP 5: Eventueel vervangen van de meetlocatie in het geval het originele meetpunt niet tot de doelpopulatie behoort

> [zie protocol sfp-001-nl versie 2023.01](../2023.01/index.html)

Reviewer: ```suggestion
Zie [protocol sfp-001-nl versie 2023.01](../2023.01/index.html).
```

> ### STAP 6:subprotocol vegetatie-opname met Londo-schaal

Reviewer: ```suggestion
### STAP 6: Subprotocol vegetatie-opname met Londo-schaal
```

> [zie protocol sfp-401-nl versie 2023.03](../2023.03/index.html)

Reviewer: ```suggestion
Zie [protocol sfp-401-nl versie 2023.03](../2023.03/index.html).
```

> ### STAP 7:meten van de variabelen in de 18 meter-structuurplot

Reviewer: ```suggestion
### STAP 7: Meten van de variabelen in de 18 meter-structuurplot
```

# Overkoepelende werkwijze met verwijzing naar subprotocols

> ## Overzicht van de verschillende stappen

Author: ok

**Onderstaande beslisregels worden gehanteerd voor het al dan niet opmeten van een proefvlak**

-   de vegetatieplot maakt deel uit van een homogene, grotere vlek doelvegetatie --\> opname doen
-   de vegetatieplot maakt geen deel uit van een doelvegetatievlek --\> opname niet doen voor dit doeltype, ga naar stap 5 om te bepalen hoe je eventueel het meetpunt kan vervangen door een andere meetlocatie in de nabijheid
-   de vegetatieplot bevat naast doelvegetatie ook (een) andere vegetatietype(s) die ruimtelijk duidelijk van elkaar gescheiden zijn (bv. bos naast een heideterrein, weg/akker naast een heide)
    -   --\> opname niet doen wanneer aandeel doelvegetatie in de vegetatieplot kleiner is dan 50%,
    -   --\> opname doen wanneer doelvegetatie in de vegetatieplot gelijk of groter is dan 50%

Indien het eerder een geleidelijke overgang betreft naar een verwant vegetatietype dan wordt dit beschouwd als een onderdeel van de doelvegetatie en wordt de opname gemaakt inclusief de overgang.

Noteer steeds op welk aandeel (in %) van de plot je data betrekking heeft.
Doe dit zowel voor de vegetatieplot als de cirkelvormige structuurplot.

> Bij het schatten van de bedekkingen blijf je echter werken alsof je op een volledige plot zou doen, je gaat m.a.w. niet herrekenen voor het aandeel dat effectief doelvegetatie bevat.

Reviewer: ```suggestion
Bij het schatten van de bedekkingen van de aanwezige soorten blijf je dit doen ten opzichte van de volledige plot. Je gaat de bedekkingen dus niet herrekenen voor het aandeel dat effectief doelvegetatie bevat.
```

Bv.
in een 3 m x 3 m plot blijft 30 cm x 30 cm dus 1%, ook wanneer 1/4 van de volledige plot niet tot de doelvegetatie behoort.

Voor de structuurplot (zie stap 7) wordt gelijkaardig gewerkt en dient ook het aandeel doelvegetatie waarop de inschatting betrekking heeft vermeld te worden.

> De inschatting van de bedekking gebeurt eveneens alsof het een volledige cirkelplot betreft, m.a.w. 10 m² is 1%.

Reviewer: ```suggestion
De inschatting van de bedekking van de structuurvariabelen gebeurt eveneens alsof het een volledige cirkelplot betreft, m.a.w. 10 m² is 1%.
```

In geval van twijfel voeren we de opname uit en documenteren we de problemen in het opmerkingenveld.

### STAP 5: Vervangpunten selecteren in het geval het originele meetpunt niet tot de doelpopulatie behoort

Zie [protocol sfp-001-nl versie 2023.01](../2023.01/index.html).

### STAP 6: Vegetatie-opname maken met behulp van de Londo-schaal in de 3 m x 3m plot

Zie [protocol sfp-401-nl versie 2023.03](../2023.03/index.html).

### STAP 7: Meten van de variabelen in de 18 meter-structuurplot

Aandeel doelvegetatie in de 18 meter-plot opgeven!

Alle criteria dienen voor elk habitattype gescoord te worden.
Indien afwezig scoor je met de waarde 0.

**Verbossing**

> De 18 meter-structuurplot wordt bekeken voor wat betreft verbossingsgraad (bedekking totale kroonprojectie in %).
> Alle houtige gewassen vanaf 80cm hoogte worden meegeteld voor deze indicator.

Reviewer: ```suggestion
Om de verbossing te bepalen, schatten we de bedekking (%) van de totale kroonprojectie van alle houtige gewassen met hoogte van minstens 80 cm.
```

Bomen en struiken die vanuit een naastgelegen bos of houtkant overhangen worden niet meegerekend voor deze indicator.
Eventuele overlap tussen boom- en struiklaag wordt niet dubbel geteld.
Het gaat m.a.w. over het aandeel doelvegetatie in de structuurplot die overdekt wordt door bomen of struiken.
Hou voor de schatting van de bedekking in het achterhoofd dat 10 m² ongeveer 1% is van de 18 meter-cirkelplot (\~1017 m²).

> Schattingen tot op 1% nauwkeurig zijn redelijk haalbaar bij lage bedekkingen,maar worden lastiger naarmate de verbossingsgraad toeneemt.

Reviewer: ```suggestion
Schattingen tot op 1% nauwkeurig zijn redelijk haalbaar bij lage bedekkingen, maar worden lastiger naarmate de verbossingsgraad toeneemt.
```

Bij hogere verbossingsgraad is een schatting tot op 5% voldoende nauwkeurig.
Interpretatie via luchtfoto of een schets zijn hierbij een hulpmiddel.

> Indien een gedeelte van de plot geen doelvegetatie bevat wordt dit deel niet in beschouwing genomen voor de verbossing.
> Maar de inschatting gebeurt wel volgens dezelfde verdeling m.a.w. 10 m² blijft 1% ook al bevindt zich maar in de helft van de vegetatieplot effectief doelvegetatie.

Reviewer: ```suggestion
Indien een gedeelte van de plot geen doelvegetatie bevat, wordt dit deel niet in beschouwing genomen voor de verbossing.
Maar de inschatting gebeurt wel volgens dezelfde verdeling, m.a.w. 10 m² blijft 1% ook al bevindt zich maar in de helft van de vegetatieplot effectief doelvegetatie.
```

**Successiestadia van buntgrasvegetaties**

Vooral bij de landduinen (2330 en 2310) is de aanwezigheid van de verschillende ontwikkelingsstadia een belangrijke structuurparameter: open zand, buntgrasvegetatie, mostapijtjes, korstmosvegetatie.

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Reviewer: bijlage 1 wordt verder in protocol ook nog vermeld

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Author: good point Toon, maar hoe lossen we dit op @hansvancalster?
toevoegen als subprotocol eigenlijk best denk ik niet?

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Author: we hebben namelijk eigenlijk de protocols van zowel Londo (sfp-401) als BHM nodig
maar deze laatste zit nog niet in de repo protocolhelper denk ik???

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Reviewer 2: Ik heb eens gekeken of de parametrisatie van sfp-401-nl het mogelijk maakt om het hier nodige subprotocol te genereren. Ik vrees dat er in die versie nog onvoldoende rekening is gehouden met de mogelijkheid van een cirkelvormig proefvlak. bv bij bespreking van materialen is er sprake van vier stokken om proefvlak uit te zetten.
Er zijn verschillende mogelijkheden:

- in dit protocol gewoon zelf beschrijven hoe de opnamen in de cirkel moet gemaakt worden
- een nieuwe versie van sfp-401-nl maken die wel beter toelaat om met dit geval om te gaan
- een nieuw protocol schrijven speciaal voor de BHM schaal & cirkelproefvlak en dit als dependency toevoegen aan dit protocol

Het tweede geval is vrij complex vrees ik. Ik zou dus de eerste of de derde optie nemen.

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Author: Moet er specifiek voor een cirkelvormige plot een apart BHM protocol opgesteld worden?
Het BHM protocol kan toch ook via paramters voor plotvorm generiek gehouden worden.
Ik wil dit eventueel wel naar de repo overzetten, maar ik zou dan opteren om in dit protocol een voorlopige verwijziging te steken, die we eens sfp-BHM klaar is toevoegen als subprotocol?
Akkoord?

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Reviewer: ok

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Reviewer 2: > Moet er specifiek voor een cirkelvormige plot een apart BHM protocol opgesteld worden? Het BHM protocol kan toch ook via paramters voor plotvorm generiek gehouden worden.

`sfp-401-nl` heeft al een parameter voor plotvorm (default = "vierkant"), maar zoals ik schreef, is er in deze versie van dat protocol onvoldoende rekening gehouden met wat er moet veranderen als je hier "cirkel" van zou maken. M.a.w. bepaalde zaken zijn nog hard gecodeerd als vaste tekst die eigenlijk moet veranderen. Mijn eerste reactie was dat dit vrij complex zou zijn om aan te passen, maar ik kom hierop terug. Iemand moet dat eens rustig bekijken om in te schatten of dit te doen is of niet (zelf voorlopig even geen tijd). 

> Ik wil het BHM protocol eventueel wel naar de repo overzetten, maar ik zou dan opteren om in protocol spp-001-nl een voorlopige verwijziging te steken, die we eens sfp-BHM klaar is toevoegen als subprotocol? Akkoord?

Wat bedoel je met voorlopige verwijzing? Indien iets als "nog te ontwikkelen protocol", dan kan dat, maar ik zou niet beginnen met te verwijzen naar protocolcodes als deze nog niet in gepubliceerd zijn.

> Scoor de aanwezigheid van elk van deze fases door middel van een code uit de beheermonitoringsschaal (bijlage 1) op het niveau van de structuurplot.

Author: Voorlopige opgelost door een tabel toe te voegen in Werkwijze.Rmd en te verwijzen naar een nog te ontwikkelen protocol
![image](https://github.com/inbo/protocolsource/assets/40395174/be5cf906-c3bb-4be5-9e43-9319160c884c)
volgens mij kan de branch nu als eerste versie gemerged worden

(ref:Callunastadia) Cyclische successiestadia in 4 fasen bij Calluna vulgaris (uit Burrows, 1990)

```{r Callunastadia, fig.cap="(ref:Callunastadia)"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "Calluna_fases.png"))

```

**Aanwezigheid veenmoslaag**

Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de bedekking van alle veenmossoorten samen op het niveau van de structuurplot.
Afwezigheid ook noteren!

**Aanwezigheid dwergstruiken**

Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de bedekking van alle dwergstruiken (Gewone Dophei, Struikhei, Blauwe bosbes, Stekelbrem, Kruipbrem, Rode Dophei, Lavendelhei, Rode bosbes,...) op het niveau van de structuurplot.
Afwezigheid ook aangegeven!

**Vergrassing**

Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de gezamelijke bedekking van Bochtige smele, Pijpenstrootje, zwenkgrassen, Duinriet en Struisgrassen.
Afwezigheid ook aangeven!

**Verruiging** Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de gezamelijke bedekking van Adelaarsvaren en Braam.
Afwezigheid ook aangeven!

> **Aanwezigheid grijs kronkelsteeltje** Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de bedekking van het mos Grijs kronkelsteeltje (zie Figuur \@ref(fig:Grijskronkelsteeltje)).

Reviewer: ```suggestion
**Aanwezigheid grijs kronkelsteeltje** 

Schat d.m.v. een code van de beheermonitoringsschaal (bijlage 1) de bedekking van het mos Grijs kronkelsteeltje (zie Figuur \@ref(fig:Grijskronkelsteeltje)).
```

Afwezigheid ook aangeven!

(ref:Grijskronkelsteeltje) Enkele typische verschijningsvormen van het mos Grijs kronkelsteeltje. De glasharen in combinatie met de bladvorm zijn een duidelijk veldkenmerk.

```{r Grijskronkelsteeltje, fig.cap="(ref:Grijskronkelsteeltje)"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "campintro.jpg"))

```

### Samenvattend diagram

(ref:samenvatting) Diagram stappenplan

```{r stappenplan, fig.cap="(ref:samenvatting)"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "samenvatting_Werkwijze.png"))

> 

Reviewer: Deze diagram hoort eigenlijk bij stap 4. Zou de figuur dus bij die stap weergeven.

Author: oke
